test/test_environment.py

```diff
@@ -130,17 +130,39 @@ def test_model_server_workers(hosting):
         assert env.model_server_workers == 2
 
 
+def test_default_port(hosting):
+    with patch.dict('os.environ', {'SAGEMAKER_REGION': 'us-west-2'}):
+        env = HostingEnvironment(hosting)
+        assert env.port == '8080'
+
+
+def test_port(hosting):
+    with patch.dict('os.environ', {'SAGEMAKER_BIND_TO_PORT': '42',
+                                   'SAGEMAKER_REGION': 'us-west-2'}):
+        env = HostingEnvironment(hosting)
+        assert env.port == '42'
+
+
+def test_safe_port_range(hosting):
+    with patch.dict('os.environ', {'SAGEMAKER_SAFE_PORT_RANGE': '1234',
```

Comment:
This is going to be one string like this?

Reply:
no it will always be range 1234-5678